### PR TITLE
build(aio): render titles correctly in content pages

### LINKED ISSUE #16099: Content H1 headers are visually smaller than H2s
**I'm submitting a ...**  (check one with "x")
[x] bug report 

**Current behavior**
On all docs content pages, the H1 headers look visual smaller than H2:
screen shot 2017-04-18 at 6 34 54 pm

Notice that "The Header Editor" is smaller than "Setup to develop locally". The first one is H1 and the second one is H2.

**Expected behavior**
H1 should be visually more deminant :-)

It looks like this is because all H1 headers have their text node wrapped in a `<p>` element:

```
<h1><p>The Hero Editor</p></h1>
```

### PR DESCRIPTION
**Pull request by petebacondarwin**
If a document provided a title jsdoc tag then its h1
element was being rendered incorrectly as a markdown
paragraph.

This change renders the title as a markdown h1 tag
directly.

Fixes #16099
Closes #16136 (making it unnecessary)

### CODE REVIEW

File: aio/tools/transforms/templates/content.template.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% if doc.title %}<h1>{$ doc.title | marked $}</h1>{% endif %}
+{% if doc.title %}{$ ('# ' + doc.title.trim() | marked $}{% endif %}
```
Comment (gkalpak) [inline aio/tools/transforms/templates/content.template.html]: Missing losing `)`.

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/tools/transforms/templates/content.template.html (hunk 1 above)
Comment (petebacondarwin) [inline aio/tools/transforms/templates/content.template.html]: doh!

Comment (petebacondarwin) [inline aio/tools/transforms/templates/content.template.html]: I undid my changes to check that the e2e test failed. But then obviously didn't hit redo enough times afterwards!

Comment (petebacondarwin) [inline aio/tools/transforms/templates/content.template.html]: fixed - via amending the commit

**Review by gkalpak — APPROVED** [review]

**Pull request merged**